# Canvas dimesions.
float32 x_dim
float32 y_dim
float32 z_dim

# Canvas position.
# panda_services will convert from EulerXYZ to quaternion.
float32 x_pos
float32 y_pos
float32 z_pos
float32 x_rot
float32 y_rot
float32 z_rot



